catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducerDataAccessObject.java

```diff
@@ -242,8 +242,7 @@ protected void logIds(List<Metacard> metacards, String action) {
       LOGGER.debug(
           "content item(s) {} with id = {}",
           action,
-          metacards
```

Comment:
❓ Can the formatter be changed to keep the call to `stream()` on its own line for multiline streaming blocks? It'd be nice to reduce the files changed by what looks like an arbitrary style change.